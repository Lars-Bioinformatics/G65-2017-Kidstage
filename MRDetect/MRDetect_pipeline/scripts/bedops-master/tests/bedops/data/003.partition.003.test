chr1	33657	33687	+MA0068.1-Pax4	8.67655e-06	+	TAATGCTATCCCTCCCCCAGCCCCCCACCC
chr1	33666	33686	+MA0073.1-RREB1 1.97929e-06	+	CCCTCCCCCAGCCCCCCACC
chr1	33670	33690	+MA0073.1-RREB1 1.0924e-06	+	CCCCCAGCCCCCCACCCACT
chr1	33672	33682	+MA0079.2-SP1	5.66765e-06	+	CCCAGCCCCC
chr1	34375	34390	+MA0065.2-PPARG::RXRA	7.21085e-07	+	GGTGGGCAAAGGGCA
chr1	34377	34390	+MA0114.1-HNF4A	5.44281e-06	+	TGGGCAAAGGGCA
