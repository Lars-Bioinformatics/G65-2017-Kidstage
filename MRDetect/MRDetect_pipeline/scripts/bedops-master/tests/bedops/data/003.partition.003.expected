chr1	33657	33666
chr1	33666	33670
chr1	33670	33672
chr1	33672	33682
chr1	33682	33686
chr1	33686	33687
chr1	33687	33690
chr1	34375	34377
chr1	34377	34390
